Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -0,0 +1,344 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @file   AODProducerWorkflowSpec.cxx
+
+#include "AODProducerWorkflow/AODProducerWorkflowSpec.h"
+#include "DataFormatsFT0/RecPoints.h"
+#include "DataFormatsITS/TrackITS.h"
+#include "DataFormatsTPC/TrackTPC.h"
+#include "DetectorsCommonDataFormats/NameConf.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/ConfigParamRegistry.h"
+#include "Framework/DataTypes.h"
+#include "Framework/InputRecordWalker.h"
+#include "Framework/Logger.h"
+#include "Framework/TableBuilder.h"
+#include "Framework/TableTreeHelpers.h"
+#include "GlobalTracking/MatchTOF.h"
+#include "GlobalTrackingWorkflow/PrimaryVertexingSpec.h"
+#include "Headers/DataHeader.h"
+#include "ReconstructionDataFormats/Track.h"
+#include "ReconstructionDataFormats/TrackTPCITS.h"
+#include "SimulationDataFormat/DigitizationContext.h"
+#include "SimulationDataFormat/MCCompLabel.h"
+#include "SimulationDataFormat/MCEventHeader.h"
+#include "SimulationDataFormat/MCTruthContainer.h"
+#include "Steer/MCKinematicsReader.h"
+#include "TMath.h"
+#include <map>
+#include <vector>
+
+using namespace o2::framework;
+
+namespace o2
+{
+namespace aodproducer
+{
+
+void AODProducerWorkflowDPL::findMinMaxBc(gsl::span<const o2::ft0::RecPoints>& ft0RecPoints, gsl::span<const o2::dataformats::TrackTPCITS>& tracksITSTPC)
+{
+  for (auto& ft0RecPoint : ft0RecPoints) {
+    uint64_t bc = ft0RecPoint.getInteractionRecord().orbit * o2::constants::lhc::LHCMaxBunches + ft0RecPoint.getInteractionRecord().bc;
+    if (minGlBC > bc)
```

Comment:
The coding and naming conventions ask to put braces even around single line bodies. Can you please change this consistently throughout this PR? (There is an ongoing campaign to fix this in all of O2 but better to avoid these changes in the first place).

Reply:
Sure, I will add the brackets